src/System.Runtime.InteropServices/tests/System.Runtime.InteropServices.Tests.csproj

```diff
@@ -141,5 +133,20 @@
     <Compile Include="System\Runtime\InteropServices\TypeLibTypeAttributeTests.cs" />
     <Compile Include="System\Runtime\InteropServices\TypeLibVarAttributeTests.cs" />
     <Compile Include="System\Runtime\InteropServices\TypeLibVersionAttributeTests.cs" />
+    <Compile Include="System\Runtime\InteropServices\Marshal\GetHINSTANCETests.cs" />
+    <Compile Include="System\Runtime\InteropServices\Marshal\ReadWrite\ByteTests.nonnetstandard.cs" />
```

Comment:
nit: could you move this down to other System\Runtime\InteropServices\Marshal\ReadWrite\